help/destinations/catalog/marketing-automation/rainfocus.md

```diff
@@ -0,0 +1,157 @@
+---
+title: RainFocus
+description: Learn how to use the RainFocus Destination Connector to syncronize audience profiles with the RainFocus Global Attendee Profile .
+---
+
+# RainFocus {#rainfocus-destination}
+
+## Overview {#overview}
+
+The RainFocus destination connector allows organizations to stream customer profiles from the Adobe Experience Platform into the RainFocus platform in order to create/update Attendee Profiles. 
+
+>[!IMPORTANT]
+>
+>The destination connector and documentation page are created and maintained by the RainFocus team. For any inquiries or update requests, please contact them directly at clientcare@rainfocus.com or visit the RainFocus [Help Center](https://help.rainfocus.com/hc/en-us).
+
+## Use cases {#use-cases}
+
+To help you better understand how and when you should use the RainFocus destination, here are sample use cases that Adobe Experience Platform customers can solve by using this destination.
+
+### Use case #1 {#use-case-1}
+
+A large enterprise technology company is due to open registration for its upcoming global expo and would like to push customer profiles to RainFocus in order to streamline the registration process.
+
+### Use case #2 {#use-case-2}
+
+A financial services brand is due to host a series of roadshows targeting new and existing customers. They have a series of audience segments with target customers in Adobe Experience Platform. Using the RainFocus Destination Connector they are able to easily send those profiles to RainFocus for activation.
+
+## Prerequisites {#prerequisites}
+  
+* RainFocus API Profile with OAuth (Global)
```

Comment:
For all these requirements, is there associated documentation on the RainFocus side to link to?